Cargo.toml

```diff
@@ -27,12 +27,12 @@ serde_repr.workspace = true
 serde.workspace = true
 
 tokio = { workspace = true, features = [
-  "macros",
-  "rt-multi-thread",
-  "sync",
-  "fs",
-  "time",
-  "full",
+    "macros",
```

Comment:
We recommend using two spaces as format. Otherwise, these files will be changed very frenquently